pubsub/README.rst

```diff
@@ -139,6 +139,25 @@ any metrics agent that implements the same interface as ``MetricsAgent``
   in a single request.
 
 
+If you prefer pull-based metrics like Prometheus you will be pleased to know that
+the subscriber records Prometheus metrics of the form ``gcloud_aio_<metric>``,
+which will be ignored if you don't use Prometheus and will be automatically
+included if you do:
+
+- ``pubsub_consume_total`` (labels: ``outcome = {'succeeded', 'cancelled',
```

Comment:
Note: I think you renamed these to eg. `pubsub_subscriber_*`, need to make sure the readme matches code.